src/main/java/Deadline.java

```diff
@@ -0,0 +1,27 @@
+public class Deadline extends Todo {
+    String by;
+
+    public String getBy() {
+        return by;
+    }
+
+    public void setBy(String by) {
+        this.by = by;
+    }
+
+    Deadline(String description, String by) {
+        super(description);
+        setBy(by);
+
```

Comment:
Please be consistent with your spacing! ie. most of your code has the closing brace on the immediate next line, but this one has an additional space